tests/integration/components/render-popover-new-api-test.js

```diff
@@ -18,6 +18,15 @@ function openModal(context, componentSpec, options={}) {
   return Ember.run(() => context.container.lookup('service:popover').openModal(componentName, options));
 }
 
+function openPopoverModal(context, componentName, options={}) {
+  return Ember.run(() => {
+    let popoverService = context.container.lookup('service:popover');
+    let { popoverSpec, close } = popoverService.buildPopoverSpec(componentName, document.body, options);
```

Comment:
In this PR what does a popoverSpec mean, I see that it is an object, but what does it achieve in this context?

Reply:
I added documentation that attempts to address this questions: https://github.com/Addepar/ember-widgets/pull/292/files#diff-4509ec26f384db2ae721a20cf7990a8fR62

The name `popoverSpec` is a bit unfortunate and perhaps we can revisit it. A `popoverSpec` is an object which is eventually rendered by the `{{render-popover}}` component. It is effectively a wrapper around a component name, arguments to a component, a method to close the component, and a destination element. Basically everything you need to render something with `{{render-popover}}`